module/programs/helper.js

```diff
@@ -1218,6 +1222,83 @@ module.exports = class ProgramsHelper {
       }
     });
   }
-};
+  /**
+   * List Program using organization id.
+   * @method
+   * @name listOrganizationPrograms
+   * @query {String} type - Assets type (program/solutions).
+   * @returns {Object} - Details of the program under the organization.
+   */
+
+  static listOrganizationPrograms(bodyData, queryData) {
```

Comment:
@praveenKDass Same comment as listOrganizationSolutions

Reply:
removed listOrganizationSolutions function and used  list function directly